modules/axios/plugin.js

```diff
@@ -118,6 +136,23 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if(options.debug) { %>
+  axios.interceptors.request.use(config => {
+    debug('[@nuxtjs/axios] Request:', config)
+    return config
+  }, error => {
+    debug('error', '[@nuxtjs/axios] Error:', error)
+    return Promise.reject(error)
+  });
+  axios.interceptors.response.use(config => {
+    debug('[@nuxtjs/axios] Response:', config)
+    return config
+  }, error => {
+    debug('error', '[@nuxtjs/axios] Error:', error)
+    return Promise.reject(error)
```

Comment:
Yes, I can confirm. The reason I added this debug property is because I have no clue why the request in my middleware is giving a 404 and it's almost impossible to find out at the moment.

As I was still busy with this instead of solving my original problem, I can confirm I still get a '404 Page not found' error in my project ;)